index.html

```diff
@@ -180,28 +226,36 @@ <h1>SKILLS</h1>
 
       </div>
     </div>
+  </div>
   </section>
 
   <!-- Form section -->
   <section class="section-form">
+    <div class="form-part">
     <div class="form-paragraph">
       <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
     </div>
     <form action="https://formspree.io/f/mrgjjrrq" method="post">
-      <input name="Name" id="name" type="text" maxlength="30" required placeholder="Full name"
-        class="input-field">
+      <input name="Name" id="name" type="text" maxlength="30" required placeholder="Full name" class="input-field">
+     <div class="desktop-input">
+      <input name="first-Name" id="first-Name" type="text" maxlength="30" required placeholder="First Name" class="input-field">
+      <input name="last-Name" id="last-Name" type="text" maxlength="30" required placeholder="Last Name"  class="input-field">
+     </div>
       <input name="Email" id="email" type="email" placeholder="Email address" required class="input-field">
       <textarea name="message" id="message" cols="30" rows="10" required maxlength="500" placeholder="Enter text here"
         class="text-area"></textarea>
       <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
-      <button type="submit" value="submit">Get in touch</button>
+     <div class="form-button">
+        <button type="submit" value="submit">Get in touch</button>
+     </div>
     </form>
-
+  </div>
     <!-- Social Media links -->
     <div class="social-media-icon">
       <ul>
         <li><a href="https://twitter.com/ojesanmi_oyin"> <img src="./images/twitter.png" alt=""></a></li>
-        <li><a href="https://www.linkedin.com/mwlite/in/elizabeth-oyinlade-ojesanmi-0702aa16a"> <img src="./images/Linkedin-icon.svg" alt=""></a></li>
+        <li><a href="https://www.linkedin.com/mwlite/in/elizabeth-oyinlade-ojesanmi-0702aa16a"> <img
+              src="./images/Linkedin-icon.svg" alt=""></a></li>
         <li><a href="#"><img src="./images/Messenger.png" alt=""></a></li>
```

Comment:
- [ ] I think it would be better if you use the background image as it is missing, `get me touch` button should be at the left not in the middle, and before social media icons there is a border-buttons kindly implement as well
- [ ] kindly remove the email on the desktop version
### your design
![his](https://user-images.githubusercontent.com/81727988/171651767-5a8ca707-1a40-48e6-a066-e5e04214df37.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/171651842-986160ed-28d1-465e-804f-38071ce75afd.png)

Reply:
- This issue is well resolved :heavy_check_mark: